examples/generalized-compute/interactive-3d-slicer-with-extensions/README.md

```diff
@@ -0,0 +1,26 @@
+# Interactive Container Example - Interactive 3D Slicer with Extensions
+<br/>
+
+### **Description**
+
+This example provides files that can be used with the Rhino Health Generalized Compute capability to remotely run an interactive 3D Slicer session to view and/or annotate DICOM data on a Rhino Client. It also includes several extensions that allow a user to save their output as DICOM images instead of NIfTI
```

Comment:
```suggestion
This example provides files that can be used with the Rhino Health Generalized Compute capability to remotely run an interactive 3D Slicer session to view and/or annotate DICOM data on a Rhino Client. It also includes several extensions that allow a user to save their output as DICOM images instead of NIfTI.
```